{{cookiecutter.github_repo}}/scripts/build-run/build.sh

```diff
@@ -16,9 +16,9 @@ while getopts ":dp" opt; do
         p)
           if [ `uname -s` = "Linux" ]
           then
-            sudo docker-compose build
+            sudo docker-compose build api
           else
-            docker-compose build
+            docker-compose build api
```

Comment:
@BrianHGrant Did you find that adding the service name here was necessary? (I had run into trouble with not including it in my own testing).

Reply:
I feel it would in some situations, though don't have 100% info, but seems to make sense to have?

Reply:
Agree. I support having the service name in like you have here.